core/src/test/java/org/testcontainers/utility/TestcontainersConfigurationTest.java

```diff
@@ -46,6 +46,10 @@ public void shouldReadReuseFromEnvironmentOnly() {
 
         environmentProperties.setProperty("testcontainers.reuse.enable", "true");
         assertTrue("reuse enabled", newConfig().environmentSupportsReuse());
+
+        environmentProperties.setProperty("ryuk.container.image", " testcontainersofficial/ryuk:0.3.0 ");
+        assertEquals("Whitespaces still exists in image property", "testcontainersofficial/ryuk:0.3.0",newConfig().getRyukImage());
```

Comment:
I think this should probably be something like:
```suggestion
        assertEquals("trailing whitespace is removed from image name property", "testcontainersofficial/ryuk:0.3.0",newConfig().getRyukImage());
```

Reply:
Isn't that first String parameter is error message on test failure ? Then I suggest to make it 
`trailing whitespace was not removed from image name property`  what's your opinion on that one ?